ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -181,6 +181,15 @@ literals:
 - zone_air_relative_humidity_sensor
 - zone_air_deadband_temperature_setpoint
 - zone_air_temperature_sensor
+- outside_air_co2_concentration_sensor
+- outside_air_co_concentration_sensor
+- outside_air_so2_concentration_sensor
+- outside_air_no2_concentration_sensor
+- outside_air_no_concentration_sensor
+- outside_air_h2s_concentration_sensor
+- outside_air_ozone_level_sensor
```

Comment:
Ozone measured in meters? `level` inherits from distance, not concentration or density. Please correct.

Reply:
Yes Thanks